packages/hub/readme.md

```diff
@@ -39,13 +39,15 @@ $ yarn hub:watch (will rebuild app on file change)
 
 ### Interacting with the hub from a browser
 
+**NOTE**: it is important to define a unique `HUB_ADDRESS` if others are running the hub locally since all hub instances use the same Firebase realtime database.
```

Comment:
This could be clearer. Perhaps

> **NOTE**: Running this package makes a connection to a shared external Firebase database. So, to avoid colliding with other developers also testing this package, set the environment variable `HUB_ADDRESS` to one that is not being used by any other developer for testing or local development purposes.

Reply:
Addressed in 04005dd79